Source/player.cpp

```diff
@@ -929,6 +929,7 @@ void NextPlrLevel(int pnum)
 
 	if (sgbControllerActive)
 		FocusOnCharInfo();
+		CalcPlrInv(pnum, TRUE);

```

Comment:
Misleading identation

Reply:
yeah that was my fault :(

Will be fixed at next aligment